core/node/consensus/src/en.rs

```diff
@@ -99,6 +105,18 @@ impl EN {
                 .wrap("BatchStore::new()")?;
             s.spawn_bg(async { Ok(runner.run(ctx).await?) });
 
+            let (attestation_status, runner) = {
+                AttestationStatusRunner::init(
+                    ctx,
+                    Box::new(MainNodeAttestationStatus(self.client.clone())),
+                    time::Duration::seconds(5),
+                    genesis_hash,
+                )
+                .await
+                .wrap("AttestationStatusRunner::init()")?
+            };
+            s.spawn_bg(async { Ok(runner.run(ctx).await?) });
```

Comment:
wrap() ?

Reply:
IIRC all these runners return anyhow::Result

Reply:
then context

Reply:
I know, wanted to use it but in a lot of cases there was nothing, I thought it was deliberate

Reply:
if it is missing, then someone (me) was too lazy to add it, or there was enough context already (for example, in simple wrappers/helper functions)

Reply:
Maybe, it looks pretty consistent ([1](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/en.rs#L51), [2](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/en.rs#L96), [3](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/en.rs#L101), [4](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/en.rs#L106), [5](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/mn.rs#L46), [6](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/mn.rs#L51), [7](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/mn.rs#L62), [8](https://github.com/matter-labs/zksync-era/blob/dbd481e425b6c87d1e03c732e3db43a8541d1ac5/core/node/consensus/src/mn.rs#L74)). 

The only reason this runner should fail is if the genesis returned from the API changed, which has a good description.

Reply:
yes, consistently lazy.